examples/data-router/src/app.tsx

```diff
@@ -29,31 +29,31 @@ import "./index.css";
 let router = createBrowserRouter([
   {
     path: "/",
-    Component: Layout,
```

Comment:
These properties are now available in v6.9.0
https://github.com/remix-run/react-router/releases/tag/react-router%406.9.0